src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.builds

```diff
@@ -9,6 +9,10 @@
       <OSGroup>Windows_NT</OSGroup>
       <TargetGroup>netcore50aot</TargetGroup>
```

Comment:
Otherwise, you would be signing up for compat with already shipped .NET Native version - extra work, little value.